docs/source/remote_data.rst

```diff
@@ -145,6 +146,106 @@ year relative to today.
     f = web.DataReader('F', 'robinhood')
     f.head()
 
+
+.. _remote_data.alphavantage
+
+AlphaVantage
+============
+
+`AlphaVantage <https://www.alphavantage.co/documentation>`__ provides realtime
+equities and forex data. Free registration is required to get an API key. 
+
+Historical Time Series Data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Through the 
+`AlphaVantage <https://www.alphavantage.co/documentation>`__ Time Series
+endpoints, it is possible to obtain historical equities data for individual
+symbols. The following endpoints are available:
+
+* ``av-daily`` - Daily Time Series
+* ``av-daily-adjusted`` - Daily Time Series (Adjusted)
+* ``av-weekly`` - Weekly Time Series
+* ``av-weekly-adjusted`` - Weekly Time Series (Adjusted)
+* ``av-monthly`` - Monthly Time Series
+* ``av-monthly-adjusted`` - Monthly Time Series (Adjusted)
+
+.. ipython:: python
+
+    import os
+    from datetime import datetime
+    import pandas_datareader.data as web
+
+    f = web.DataReader("AAPL", "av-daily", start=datetime(2017, 2, 9),
+                       end=datetime(2017, 5, 24),
+                       access_key=os.getenv('ALPHAVANTAGE_API_KEY'))
+    f.loc["2017-02-09"]
+
+The top-level function ``get_data_av`` is also provided. This function will
```

Comment:
I don't really like get_data_av -- how could a new user know what av is?  Maybe just the long get_data_alphavantage? Autocomplete is pretty good.